chat-client/src/client/mynahUi.ts

```diff
@@ -450,11 +452,11 @@ export const createMynahUi = (
         return tabId ? mynahUi.getAllTabs()[tabId]?.store : undefined
     }
 
-    const createTabId = (needWelcomeMessages: boolean = false, chatMessages?: ChatMessage[]) => {
```

Comment:
Why did we remove those 2 params from here?

Reply:
Ok, I see what happened to `chatMessages` now, but the question still remains for `needWelcomeMessages`.